.github/workflows/generate.yml

```diff
@@ -0,0 +1,49 @@
+name: Code Generation
+
+on:
+  push:
+    branches: ["main"]
+
+  workflow_dispatch:
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: frenck/action-setup-yq@v1
+      - name: Downgrade OAS
+        run: bash scripts/downgrade-oas-version.sh
+  build:
+    needs: [ setup ]
+    runs-on: ubuntu-latest
+    strategy:
+            matrix:
+                target:
+                    - go-gin-server
+                    - python-flask 
```

Comment:
Any others we should consider adding?

Reply:
Well, our dummy server is in node? And I hear a lot of industry uses c#. But do we know if anyone's using this?